compiler/x/codegen/X86Ops.ins

```diff
@@ -615,6 +615,50 @@ INSTRUCTION(VBROADCASTSDZmmXmm, vbroadcastsd,
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM),
             PROPERTY1(IA32OpProp1_XMMSource | IA32OpProp1_XMMTarget | IA32OpProp1_SIMDSingleSource),
             FEATURES(X86FeatureProp_EVEX512Supported | X86FeatureProp_EVEX512RequiresAVX512F)),
+INSTRUCTION(VPBROADCASTBRegReg, vpbroadcastb,
+            BINARY(VEX_L128, VEX_vNONE, PREFIX_66, REX__, ESCAPE_0F38, 0x78, 0, ModRM_RM__, Immediate_0),
+            PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM),
+            PROPERTY1(IA32OpProp1_XMMSource | IA32OpProp1_XMMTarget | IA32OpProp1_SIMDSingleSource),
+            FEATURES(X86FeatureProp_VEX128Supported | X86FeatureProp_VEX128RequiresAVX2 | X86FeatureProp_VEX256Supported | X86FeatureProp_VEX256RequiresAVX2 |
+                     X86FeatureProp_EVEX128Supported | X86FeatureProp_EVEX128RequiresAVX512BW | X86FeatureProp_EVEX128RequiresAVX512VL |
+                     X86FeatureProp_EVEX256Supported | X86FeatureProp_EVEX256RequiresAVX512BW | X86FeatureProp_EVEX256RequiresAVX512VL |
+                     X86FeatureProp_EVEX512Supported | X86FeatureProp_EVEX512RequiresAVX512BW)
+            ),
+INSTRUCTION(VPBROADCASTBRegMaskReg, vpbroadcastb,
+            BINARY(EVEX_L128, VEX_vNONE, PREFIX_66, REX__, ESCAPE_0F38, 0x7a, 0, ModRM_RM__, Immediate_0),
```

Comment:
Not saying you need to change this, but since this opcode has the same mneumonic as the other instructions, there might be some confusion regarding the operand types. Maybe change suffix to GPR? `RegMaskGPR`?

Reply:
Changed as suggested. Thanks.